pyproject.toml

```diff
@@ -48,17 +48,19 @@ Changelog = "https://github.com/robbievanleeuwen/section-properties/releases"
 
 [tool.poetry.dependencies]
 python = ">=3.9.0,<3.12"
-numpy = "^1.26.0"
+numpy = "^1.25.2"
```

Comment:
Could you put a comment here stating the reason for the downgrade so I can remember to check this if `numba` allows further `numpy` updates.

```
e.g. numpy = "^1.25.2"  # numba requires numpy <1.26
```